tests/ltp/patches/getxattr02.patch

```diff
@@ -0,0 +1,65 @@
+diff --git a/testcases/kernel/syscalls/getxattr/getxattr02.c b/testcases/kernel/syscalls/getxattr/getxattr02.c
+index dca6b13be..891308dd4 100644
+--- a/testcases/kernel/syscalls/getxattr/getxattr02.c
++++ b/testcases/kernel/syscalls/getxattr/getxattr02.c
+@@ -38,6 +38,10 @@
+  *    return -1 and set errno to ENODATA
+  */
+ 
++/* Currently xattr is not enabled while mounting root file system. Patch is
++ * mount root file system with xattr enabled and then use  it for the test.
++ */
++
+ #include "config.h"
+ #include <sys/types.h>
+ #include <sys/stat.h>
+@@ -51,7 +55,7 @@
+ #include <stdlib.h>
+ #include <string.h>
+ #ifdef HAVE_SYS_XATTR_H
+-# include <sys/xattr.h>
++#include <sys/xattr.h>
```

Comment:
not needed

Reply:
@letmaik I asked this to be fixed since #include looks more proper than # include. Don't you aggree? Do we really need to take it back?